### PR TITLE
readme: update installation instructions for Arch Linux

### PR DESCRIPTION
**Pull request by orhun**
`wikiman` is moved to the community repository: https://archlinux.org/packages/community/any/wikiman/

This PR also splits the Arch Linux and Manjaro instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,21 +16,38 @@ Follow [these](#additional-documentation-sources) instructions to enable optiona
 
 ## Installation
 
-### Arch Linux / Manjaro ([AUR](https://aur.archlinux.org/packages/wikiman/))
+### Arch Linux
+
+**Wikiman** is available in the [community repository](https://archlinux.org/packages/community/any/wikiman/) and can be installed via [pacman](https://wiki.archlinux.org/title/Pacman):
```
Comment (filiparag) [inline README.md]: I would reword this to be more consise, akin to [FreeBSD](https://github.com/filiparag/wikiman#freebsd) one:
```markdown
Install from Arch Linux's [community](https://archlinux.org/packages/community/any/wikiman/) repository:
```

File: README.md [hunk 2]
```diff
@@ -16,21 +16,38 @@ Follow [these](#additional-documentation-sources) instructions to enable optiona
...
 ```bash
 sudo pacman -U wikiman*.pkg.tar.zst
 ```
 
+### Manjaro
+
+Similar intructions with [Arch Linux](#arch-linux) apply.
+
+```bash
+yay -Syu wikiman
```
Comment (filiparag) [inline README.md]: I'm not sure whether keeping instructions for Manjaro using `yay` makes sense if the package got yanked from AUR and is not in their repositories. This command currently doesn't work on Manjaro.

**Review by filiparag — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (orhun) [reply README.md]: I'm not familiar with Manjaro, what do you suggest here?

File: README.md (hunk 1 above)
Comment (orhun) [reply README.md]: 49804299ba104136ff5eea065d8a67310ebdae06

File: README.md (hunk 2 above)
Comment (filiparag) [reply README.md]: I gave it some thougth and think that the best course of action is to drop *Manjaro* from *Arch Linux* heading and mention it below like:
> If you are running Manjaro or another Arch-based distribution, download the latest *.pkg.tar.zst* package from [Releases](https://github.com/filiparag/wikiman/releases/latest/) tab.
> ```sh
> sudo pacman -U wikiman*.pkg.tar.zst
> ```
> Follow [these](https://github.com/filiparag/wikiman#installing-additional-sources) instructions to add Arch Wiki as a source.

If you agree, I can commit these changes and merge the PR.

Comment (orhun) [reply README.md]: Sounds good to me.

**Review by filiparag — APPROVED** [review]
